AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -994,6 +994,9 @@ lequipe.fr#@#.text_ad
 lequipe.fr#@#.text_ads
 lequipe.fr#@#.text-ads
 lequipe.fr#@#.text-ad-links
+||cdn.freewebcart.com/wp-content/cache/min/*/wp-content/plugins/wp-jquery-update-test/assets/jquery*.js
+||freewebcart.b-cdn.net/wp-content/plugins/wp-jquery-update-test/assets/jquery*.js
+||freewebcart.b-cdn.net/wp-includes/js/jquery/*
```

Comment:
I'm not sure if it's a good idea to block these scripts, I think that it may breaks some website functionality.
For instance, maybe it's not a big issue, but it seems that button to hide "Table of Contents" is broken.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/92120494-67a92780-edf9-11ea-853d-4cf2fa820ace.png)

</details>

What about something like this for products which support scriptlets:
`freewebcart.com#%#//scriptlet("abort-on-property-write", "ai_front")`

And for example for content blockers:
```
@@||freewebcart.com^$generichide
@@||contextual.media.net/dmedianet.js$domain=freewebcart.com
@@||freewebcart.b-cdn.net/wp-content/plugins/ad-inserter/*/*.js$domain=freewebcart.com
@@||freewebcart.b-cdn.net/wp-content/plugins/ad-inserter/images/ads.png$domain=freewebcart.com
@@||google-analytics.com/analytics.js$domain=freewebcart.com
@@||ib.adnxs.com^$image,domain=freewebcart.com
@@||router.infolinks.com/dyn/apn-usync$image,domain=freewebcart.com
```

And maybe to annoyances filter to skip a counter:
`freewebcart.com#%#//scriptlet("adjust-setInterval", "counter", "", "0.02")`
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/92121492-9ffd3580-edfa-11ea-9ba1-68ba478ef2fa.png)

</details>

Reply:
Cool, thanks.
I would add setInterval to other similiar rules in tweaks though.